python/ray/tests/test_runtime_env.py

```diff
@@ -865,6 +865,59 @@ def test_runtime_env_interface():
     assert runtime_env.to_dict() == {}
 
 
+def test_runtime_env_config(start_cluster):
```

Comment:
Do we support set config in ray client? If we already supported, can we also add ray_client test?

Reply:
The `start_cluster` fixture parametrizes over both Ray client and non-Ray-client, so we should be good here!